### PR TITLE
fix: Parser added for SecureAuth

### PR DESCRIPTION
**Pull request by bparmar-splunk**

### CODE REVIEW

**Comment by mateuszpierzchala-splunk** [comment]
Looks good for me.

**Review by mateuszpierzchala-splunk — APPROVED** [review]

File: tests/test_secureauth_idp.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    host = "{}-{}".format(random.choice(setup_wordlist), random.choice(setup_wordlist))
+
+    dt = datetime.datetime.now()
+    iso, bsd, time, date, tzoffset, tzname, epoch = time_operations(dt)
+
+    # Tune time functions
+    epoch = epoch[:-3]
+
+    mt = env.from_string(
+        '{{ mark }} {{ iso }} {{ host }} SecureAuth2 2928 ID90020 [SecureAuth@27389 UserAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36" UserHostAddress="10.1.2.2" RequestID="d2014021-4e06-45c6-a580-346e12346b60" Realm="SecureAuth2" Appliance="SecureAuth05VM.domain.com" Company="National Title Group Inc" Version="9.2.0.85" PEN="27389" HostName="10.1.2.2" Category="AUDIT" Priority="4" EventID="90020"] Application - Begin request'
```
Comment (rjha-splunk) [inline tests/test_secureauth_idp.py:27]: annymise the ips here

**Review by rjha-splunk — COMMENTED** [review]
Please anonymise the log message

File: tests/test_secureauth_idp.py (hunk 1 above)
Comment (bparmar-splunk) [reply tests/test_secureauth_idp.py]: Done

**Pull request merged**